ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/stack/StackComponentView.kt

```diff
@@ -632,21 +819,27 @@ private fun previewBadge(alignment: TwoDimensionalAlignment): BadgeStyle {
                     overrides = null,
                 ),
             ),
-            dimension = Dimension.Horizontal(alignment = VerticalAlignment.CENTER, distribution = START),
+            dimension = Dimension.Vertical(alignment = HorizontalAlignment.CENTER, distribution = CENTER),
```

Comment:
Made this change in the preview to properly center the content of the badge. I think the backend should also return the same style, but will test this again once it has been published to the dashboard.